soh/soh/Enhancements/randomizer/context.cpp

```diff
@@ -1,17 +1,58 @@
 #include "context.h"
 #include "static_data.h"
+#include "soh/OTRGlobals.h"
+#include "soh/Enhancements/item-tables/ItemTableManager.h"
 #include "3drando/shops.hpp"
-#include "3drando/dungeon.hpp"
+#include "dungeon.h"
+#include "trial.h"
+#include "entrance.h"
+#include "settings.h"
+#include "rando_hash.h"
 
+#include <fstream>
 #include <spdlog/spdlog.h>
 
 namespace Rando {
 std::weak_ptr<Context> Context::mContext;
 
 Context::Context() {
+    for (auto& location : StaticData::GetLocationTable()) {
+        mSpoilerfileCheckNameToEnum[location.GetName()] = location.GetRandomizerCheck();
+    }
+    mSpoilerfileCheckNameToEnum["Invalid Location"] = RC_UNKNOWN_CHECK;
+    mSpoilerfileCheckNameToEnum["Link's Pocket"] = RC_LINKS_POCKET;
+
+    for (auto& item : StaticData::GetItemTable()) {
+        // Easiest way to filter out all the empty values from the array, since we still technically want the 0/RG_NONE
+        // entry
+        if (item.GetName().english.empty())
```

Comment:
I believe our new coding standards require brackets for all if statements even if they're a single line.

Reply:
Good Catch! This code probably came from somewhere else via copy paste and was already like that so I didn't catch it.

Reply:
Although now that I'm looking closer at it, this is no longer a change in this PR post rebase, as it was in the previous rando-gap-bridging PR. So this will happen as part of a separate "tidy-up" PR, alongside some of the stuff from your other comments.